shared/actions/config/index.js

```diff
@@ -21,6 +43,7 @@ export function getConfig (): AsyncAction {
           }
 
           dispatch({type: Constants.configLoaded, payload: {config}})
+          switchTabs(dispatch, getState)
```

Comment:
you can call dispatch(switchTabs) instead, which follows our convention a bit more.
also :sheep: maybe name this something else since it's close to the switchTab action?